README.md

```diff
@@ -437,14 +447,15 @@ Here's the wild card rule for almost all event types taken from `claims-processi
     new CloudWatchLogGroup(allEventsLogGroup),
     new LambdaFunction(createMetricsLambdaFunction),
   ],
-})
-```   
+});
+```
 
-These metrics are used to create a CloudWatch dashboard to present business KPIs and metrics for Lambda function, API Gateway APIs, event rules, and state machines. 
+These metrics are used to create a CloudWatch dashboard to present business KPIs and metrics for Lambda function, API Gateway APIs, event rules, and state machines.
 
 In AWS Console, navigate to CloudWatch > Dashboards > Claims-Processing-Dashboard to review this dashboard.
 
 ## Cleanup
+
```

Comment:
Let's add some verbiage here to help the audience understand how "Clear All Data" works.

Reply:
Updated